dist/index.html

```diff
@@ -8,8 +8,27 @@
     <script src="../src/index.js"></script>
   </head>
   <body>
-    <div class="container">
-      <h1 class="center">Hello World!</h1>
+    <div class="parent-container">
+      <!-- <h1 class="center">Hello World!</h1> -->
+      <div class="navigation-container">
+        <button type="button" class="nav">Home</button>
+        <button type="button" class="nav">Projects</button>
+        <button type="button" class="nav">About Me</button>
+        <button type="button" class="nav">Contact</button>
+      </div>
+      <span class="perks">
```

Comment:
**Lines 19-21:**

`<span>` tags are generally used to format **sub-sections** of text in-line, kind of like this:

```
<p>I am a paragraph with lots of sentences. Every once in a while, <span style="color: green;">some of my text is green</span>, but usually, it is just black. Sometimes I get a little wild and <span style="color: pink;">turn my text pink</span>, but I <span style="color: green;">rarely</span> do that.</p>
```

But we don't often use them to wrap an entire body of text. Instead, just go with a `<div>` tag.